src/Carbon/Lang/bn_BD.php

```diff
@@ -9,10 +9,6 @@
  * file that was distributed with this source code.
  */
 
-/*
- * Authors:
- * - Ankur Group, http://www.ankurbangla.org, http://www.bengalinux.org Taneem Ahmed, Jamil Ahmed taneem@bengalinux.org, jamil@bengalinux.org
```

Comment:
Both those URLs return 404s.

Reply:
But just because the URLs don't work, there's no reason to remove the groups/names?

Reply:
Re-added